README.md

```diff
@@ -163,7 +163,7 @@ language_default: es
 languages: en|es
 ```
 
-Most of the navigation, buttons, and general wording throughout orchid has been pulled out into `locales/en.yml`.  Copy that file to match the other language(s) your app will support, for example: `locales/es.yml`.  Translate each entry of the yaml file.  You may toggle between languages in the application and view the language differences. **This file must exist for every language your app config specifies, though you may leave it blank.**
+Most of the navigation, buttons, and general wording throughout orchid has been pulled out into `locales/en.yml`.  Copy that file to match the other language(s) your app will support, for example: `locales/es.yml`.  Translate each entry of the yaml file.  You may toggle between languages in the application and view the language differences. **This file must exist for every language your app config specifies, though you may leave it blank.**  Double check that you have changed the key `en` at the top of the file, or else rails will not be able to find the appropriate language text for your application.
```

Comment:
:thinking: Though the "you may leave it blank" part and needing the key at the top to match is kind of self-contradicting. I'm assuming the file has to exist or there are errors during initialization, whereas the key has to be there when actually switching locale or there are errors at that point. Is that right?

Reply:
Well, so if you tell it to use `es` and it has no `es` key in a locales file, rails will throw an error saying that language is not allowed.  But if it does find the file, then it's happy enough to start.  At that point, if you had something like `t "search.title", default: "Search"`, and there was no matching `"search.title"` in the `es.yml` file, rails will pull the default "Search" instead.  If you don't put in a default value, then it will display something like "search title."

It's kinda strange, it took me a bit of testing to figure out how all the fallbacks work for the various yaml files.  The process I've figured out is, crudely:

rails app en.yml -> orchid app en.yml -> default value -> the name of the requested translation

Reply:
I'm just worried about avoiding errors altogether. Would it be simpler to say in the documentation here to at least have the `(lang).yml` _and_ the initial `(lang):` key at the top then, because otherwise it throws the error that it can't find the language?

I guess I don't see the point in telling people they can leave the file blank when that will still lead to another error, just not at app initialization.

Reply:
Oh I see, I think I didn't realize I was literally saying to leave the file blank, but rather to leave everything after the key blank.  I'll work it to be more like your wording, since mine is definitely misleading.

Reply:
Ah, gotcha. Sounds good :+1: